### ISSUE TITLE
Movie Recipes Not Working With Latest Plex Agent

### ISSUE THREAD

**Issue by Migz93** [adamgot/python-plexlibrary#82]
It seems that any movie recipes don't function correctly if the library they're pointing at is using the new Plex Movie Agent

When running the recipe python-plexlibrary can't see any movies in my overall movie library and just lists everything as missing.

**Comment by Radikaltimes** [comment]
same here, hope that is something that can be fixed

**Comment by sysmoon14** [comment]
Yeah I'm seeing this issue too

**Comment by adamgot** [comment]
With what version of Plex did this issue start? Can you confirm that links are created and are readable? Does it work with another agent? It sounds like an issue with the Plex agent not following links, not much we can do about it in that case.

**Comment by Migz93** [comment]
It started around the 1.20.0.xxxx range but only on libraries that are using the new plex agent. Libraries that use the old agent even with the latest plex version still work fine.

The links are not created at all, plexpython can't see any movies in my main library to create links from.
Heres a pastebin of the output
https://pastebin.com/ZXY6uTSu

You can see that it lists 130 missing items that it doesn't think i have, which i can see theirs movies in there i definitely do, it just doesn't seem to be able to read the list of movies in my main library now that the library is set to use the new agent.

**Comment by sysmoon14** [comment]
There's a couple of plex forum posts about this issue.

https://forums.plex.tv/t/new-plex-movie-agent-not-populating-imdb-tmdb-id-field-in-xml/618519

https://forums.plex.tv/t/new-plex-movie-agent-does-not-contain-imdb-info/618121

It's worth noting that I believe Plex has an update to its TV Shows agent in the works that would likely cause the same issue.

**Comment by Radikaltimes** [comment]
yeah is 100% the agent the one doing that, dont think adamgot will be able to do much.

**Comment by boff999** [comment]
One of the links above has a comment that mentions using the advanced settings on a library to use the legacy agent rather than the new one.  Everything seems to work fine once this has been set, although I refreshed the metadata on each library just to be safe.

**Comment by Radikaltimes** [comment]
> One of the links above has a comment that mentions using the advanced settings on a library to use the legacy agent rather than the new one. Everything seems to work fine once this has been set, although I refreshed the metadata on each library just to be safe.

i may need to refresh the metadata .. because i still get no results

**Comment by docmeth02** [comment]
I commented here with a quick and dirty fix:
#90
